File: go/multiple_input.go
package main

import "fmt"

func main() {
	var name string
	var age int
	fmt.Print("Enter your name: ")
	fmt.Scanln(&name)
	fmt.Print("Enter your age: ")
	fmt.Scanln(&age)
	fmt.Println("Hello", name)
	fmt.Println("You are", age, "years old")
}

File: nodejs/multiple_input.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Enter your name: ", name => {
    readline.question("Enter your age: ", age => {
        console.log("Hello " + name);
        console.log("You are " + age + " years old");
        readline.close();
    });
});